lib/cli.js

```diff
@@ -99,6 +99,10 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' The TAP reporter is not available when using watch mode.');
 	}
 
+	if ((hasFlag('--watch') || hasFlag('-w')) && isCi) {
+		throw new Error(colors.error(figures.cross) + ' AVA will not run with the --watch (-w) option in CI, because CI processes should terminate, and with the --watch option, AVA will never terminate.');
```

Comment:
Proposal for a shorter version:

`' AVA not run if using watch mode in CI, as CI processes should terminate and AVA will never terminate.'`

before:
* 147 characters

after:
* 102 characters

For information, CLI error with Tap reporter : 57 characters